src/action/wallet.js

```diff
@@ -261,6 +261,25 @@ class WalletAction {
     }
   }
 
+  /**
+   * Ensure that the wallet address is non-null before navigating to the
+   * NewAddress view during onboarding.
+   * This is necessary because the wallet address may be null if neutrino
+   * has not started syncing by the time the user finishes verifying
+   * their seed.
+   * @return {undefined}
+   */
+  addInitialCoin() {
```

Comment:
Rename to `initInitialDeposit` for consistency.

Reply:
fixed